pocketimdb/movies/views.py

```diff
@@ -24,6 +24,8 @@ def get_queryset(self):
             likes=Coalesce(Count('movie_likes__like', filter=Q(movie_likes__like=Like.LIKE)), 0),
             dislikes=Coalesce(Count('movie_likes__like', filter=Q(movie_likes__like=Like.DISLIKE)), 0),
             user_liked_or_disliked=Coalesce(Sum('movie_likes__like', filter=Q(movie_likes__user=self.request.user)), 0),
+            in_user_watch_list=Count('watchlist__id', filter=Q(watchlist__user=self.request.user)),
```

Comment:
ovdje ces dobiti broj, a ideja je da dobijes True ili False i za boolean vrijednosti kada pises gledaj da ti uvijek bude tipa prefix `did_user_watch`, `is_visible` itd..

Reply:
Meni je ovo polje u serializer-u boolean. U sustini je to dosta magicno, on ako ovde nadje bilo sta tamo ce se postaviti `true`, a ako count vrati 0 on ce se postaviti tamo u serializer-u na `false`.